rust/rsc/src/rsc_tool/main.rs

```diff
@@ -166,6 +170,33 @@ async fn remove_all_jobs(db: &DatabaseConnection) -> Result<(), Box<dyn std::err
     Ok(())
 }
 
+async fn bootstrap_db(db: &DatabaseConnection) -> Result<(), Box<dyn std::error::Error>> {
+    // Get all the info from the user
+    let use_dev_key = Confirm::new("Create a test API key? (development only!)")
+        .with_default(false)
+        .prompt()?;
+    let key_desc = Text::new("What is the key used for?").prompt()?;
+    let local_store_root = Text::new("Where should local blobs be saved?").prompt()?;
+
+    // Create the API key
+    let key = if use_dev_key {
+        Some("InsecureKey".into())
+    } else {
+        None
+    };
+    add_api_key(key, key_desc, db).await?;
+
+    // Create the local blob store
+    add_local_blob_store(local_store_root, db).await?;
+
+    // Create "DBOnly blob store"
```

Comment:
Not sure what the scare quotes are for here.

Reply:
me either lol, don't remember typing them